master/internal/trial.go

```diff
@@ -395,10 +395,6 @@ func (t *trial) processScheduler(ctx *actor.Context) error {
 	case scheduler.ContainerStarted:
 		t.containers[msg.Container.ID()] = msg.Container
 
-		if err := t.pushRendezvous(ctx); err != nil {
```

Comment:
blocking: with this change if the harness websocket connects before the agent notifies the trial actor that the container has started we will never push rendezvous info.